ansible-scylla-monitoring/tasks/RedHat.yml

```diff
@@ -24,8 +24,8 @@
   become: true
 
 - name: install prerequisite python-yaml
-  shell: |
-    for i in `yum search python3|grep -i pyyaml|awk '{ print $1 }'`; do sudo yum -y install $i; done
+  dnf:
+    name: python3-pyyaml
```

Comment:
The intention here was to install a single package. It's just its name sometimes included capital letters, e.g. `python3-PyYAML`